server/src/main/java/org/apache/druid/catalog/model/table/ExternalTableDefn.java

```diff
@@ -205,6 +205,11 @@ public class ExternalTableDefn extends TableDefn
    */
   public static final String TABLE_TYPE = "extern";
 
+  /**
+   * Column type for external tables.
+   */
+  public static final String EXTERNAL_COLUMN_TYPE = "extern";
```

Comment:
What is this used for?

Reply:
This is in another part of the original PR related to external table handling, it's not used currently but future changes will